_posts/2017-12-26-2017-year-in-review.md

```diff
@@ -0,0 +1,132 @@
+
+![Flink Forward Speakers]({{ site.baseurl }}/img/blog/speaker-logos-ff2017.png)
+
+## Features and Ecosystem
+
+### Flink Ecosystem Growth
+
+Flink was added to a selection of distributions and integrations during 2017, making it easier for a wider user base to get started with Flink:
+
+-   [Official Docker image](https://hub.docker.com/r/_/flink/)
+-   [Official DC/OS and Mesos support](https://ci.apache.org/projects/flink/flink-docs-release-1.3/setup/mesos.html)
+-   [A Flink connector](https://data-artisans.com/blog/dellemc-launches-open-source-pravega-complete-apache-flink-connector) for [Pravega](http://pravega.io), Dell/EMC's streaming storage system.
+-   Uber announced AthenaX, a streaming SQL platform [powered by Apache Flink](https://data-artisans.com/blog/uber-introduces-open-source-athenax-streaming-sql-platform-apache-flink).
+-   dataArtisans announced an early access program of a SaaS product based on Apache Flink, [dA Platform 2](https://data-artisans.com/blog/da-platform-2-stateful-stream-processing-with-apache-flink-made-easier).
+
+### Feature Timeline in 2017
+
+Just in time for the end of the year, our 1.4 release [read the full release announcement](http://flink.apache.org/news/2017/12/12/release-1.4.0.html) landed in mid-December culminating 5 months of work and the resolution of more than 900 issues. This is the fifth major release in the 1.x.y series.
+
+Here's a selection of major features added to Flink over the course of 2017:
+
+![Flink Release Timeline 2017]({{ site.baseurl }}/img/blog/flink-releases-2017.png)
+
+If you take a look at [the resolved issues and enhancements for 2017 on Jira](https://issues.apache.org/jira/browse/FLINK-5016?jql=project%20%3D%20FLINK%20AND%20issuetype%20in%20(Bug%2C%20Improvement%2C%20%22New%20Feature%22)%20AND%20status%20in%20(Resolved%2C%20Closed)%20AND%20resolved%20%3E%3D%202017-01-01%20AND%20resolved%20%3C%3D%202017-12-31%20ORDER%20BY%20resolved%20ASC) you can see that the community resolved over 1,831 issues and feature additions.
+
+Regarding roadmap commitments from 2016, there is mixed news, with some items a part of current releases, others scheduled for upcoming releases and some that remain under discussion.
+
+## Looking ahead to 2018
+
+A good source of information about the Flink community's roadmap is the list of [Flink Improvement Proposals (FLIPs)](https://cwiki.apache.org/confluence/display/FLINK/Flink+Improvement+Proposals) in the project wiki. Below, we'll highlight a selection of FLIPs accepted by the community as well as some that are still under discussion.
+
+Work is already underway on a number of these features, and some will be included in Flink 1.5 at the beginning of 2018.
+
+-   **Improved BLOB storage architecture**, as described in [FLIP-19](https://cwiki.apache.org/confluence/display/FLINK/FLIP-19:+Improved+BLOB+storage+architecture) to consolidate API usage and improve concurrency.
```

Comment:
Add FLIP-24 and FLIP-23?

* **SQL Client**, as described in [FLIP-24](https://cwiki.apache.org/confluence/display/FLINK/FLIP-24+-+SQL+Client) to add a service and a client to execute SQL queries against batch and streaming tables.
* **Model Serving**, as described in [FLIP-23](https://cwiki.apache.org/confluence/display/FLINK/FLIP-23+-+Model+Serving) to add a library that allows users to evaluate machine-learning models on data streams.